README.adoc

```diff
@@ -57,23 +57,25 @@ For example, an application that provides real-time stock quotes might use SSE t
 updates from the server to the browser as soon as the server receives them. Such an application wouldn't need Web Sockets because the data travels in only one direction, and polling the server by using HTTP requests wouldn't provide real-time
 updates.
 
-This guide consists of a `frontend` application and three instances of the `system`
-service. The `system` services periodically publish messages containing their
-hostname and current system load. The `frontend` application consists of a front-end
-JavaScript client and a `bff` (backend for frontend) service. The `bff` service
-receives the messages from the `system` services and pushes the contents as events
-to the front-end client that consumes them and displays a table that contains
-each system's hostname and its periodically updating load. The following
-diagram depicts the application that is used in this guide:
+This guide consists of the `frontend` service, the `bff` (backend for frontend)
+service, and three instances of the `system` service. The `system` services
+periodically publish messages containing their hostname and current system load
+and the `frontend` service consists of a front-end JavaScript client. The `bff`
+service receives the messages from the `system` services and pushes the contents
+as events to the client in the `frontend` service that consumes them and
+displays a table that contains each system's hostname and its periodically
+updating load. The following diagram depicts the application that is used in
+this guide:
 
 image::SSE_Diagram.png[SSE Diagram, align="center"]
 
 In this guide, you will set up the `bff` service by creating an endpoint for
-clients to use to subscribe to events. You will also enable the service to read from the
-reactive messaging channel and push the contents to subscribers via SSE. After
-that, you will configure the Kafka connectors to allow the `bff` service to receive
-messages from the `system` services. Finally, you will configure the front-end
-client to subscribe to these events, consume them, and display them in the UI.
+clients to use to subscribe to events. You will also enable the service to read
```

Comment:
**change**
In this guide, you will set up the `bff` service by creating an endpoint for clients to use to subscribe to events

**to**
In this guide, you will set up the `bff` service by creating an endpoint that clients can use to subscribe to events.